assets/section-footer.css

```diff
@@ -160,46 +155,94 @@
   }
 }
 
+.footer__follow-on-shop:empty,
```

Comment:
I don't think we need this since we're already checking before we output this element: 
```liquid
{%- if shop.features.follow_on_shop? and section.settings.enable_follow_on_shop -%}
```
But maybe there scenarios in which nothing could be outputted in the `div` ?

Reply:
I thought this was to prevent the button showing on `no-js`, but you're right that it still outputs something.

So maybe I'm missing another scenario or this isn't needed.

Reply:
I just saw this pattern with the other two components and just followed it. Assumed it was there for a reason.

Reply:
```suggestion
```